backend/frontend/templates/frontend/user/board/partial_card.html

```diff
@@ -166,5 +166,17 @@
             >Start
         </button>
     {% endif %}
+    
+    {% if card.content_type_nice == "project" %}
+        {% if card.request_user_can_request_review %}
```

Comment:
Shouldn't `{% if card.request_user_can_request_review %}` be enough here? 
Ideally all the logic around figuring out if a user can request a review should be inside the `card.request_user_can_request_review` function.

Right now, most of the logic is inside the function and the rest is inside the template. It's a strange way to split things up